dungminhnguyen257/the-art-commerce
# feat: set up prisma and db connection

## Summary of changes:
- Add [prisma ORM](https://www.prisma.io/)
- Add .env file to manage environment variables
- Create admins table

## Test steps:
- Follow this to set up Postgres user: https://docs.google.com/document/d/1KIUfTRmJ8FvdHuj_hzpCB-UbkUGwZJXNaP1ZT65-LmI/edit 
- Add an `.env` file to your root directory and add the following line to the file: `DATABASE_URL="postgresql://username:password@localhost:5432/art_commerce?schema=public"`
- Run `npm install`
- Run `npx prisma db push`
- Check that admins table is in your local db
